Articles of Association.md

```diff
@@ -0,0 +1,66 @@
+1. Name
+    1. The name of the association is "Feather Association"
+2. Purpose
+    1. The purpose of the association is
+        1. The promotion of feather
+        2. The development of feather
+        3. The distribution of feather and associated software and tools
+3. Membership
```

Comment:
I don't see anything about how one becomes a voting member.

Reply:
Approval by other voting members, doing the General Assembly.

Reply:
I have made it possible for any member to be vote eligible if they are active.